docs/en/01_configuration.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Configuration
+summary: Configuration options including adding existing search, inline editing, multi-class adding, orderable rows, and configurable pagination
+icon: cogs
+---
+
+# Configuration
+
+## Add existing search
+
+The [`GridFieldAddExistingSearchButton`](api:Symbiote\GridFieldExtensions\GridFieldAddExistingSearchButton) component provides a more complete solution for adding
+existing records than a basic autocomplete. It uses the search context constructed by the model
+class to provide the search form.
+
+```php
+use Symbiote\GridFieldExtensions\GridFieldAddExistingSearchButton;
+
+$gridField->getConfig()->addComponent(GridFieldAddExistingSearchButton::create());
+```
+
+## Inline editing
+
+This example replaces the default data columns component with an inline editable one, and the
+default add new button with one that adds new records inline.
+
+```php
+use SilverStripe\Forms\GridField\GridFieldButtonRow;
+use SilverStripe\Forms\GridField\GridFieldDeleteAction;
+use SilverStripe\Forms\GridField\GridFieldToolbarHeader;
+use Symbiote\GridFieldExtensions\GridFieldAddNewInlineButton;
+use Symbiote\GridFieldExtensions\GridFieldEditableColumns;
+use Symbiote\GridFieldExtensions\GridFieldTitleHeader;
+
+$gridField = GridField::create(
+    'ExampleGrid',
+    'Example GridField',
+    $this->Items(),
+    GridFieldConfig::create()
```

Comment:
Need `use` statement for `GridField` and `GridFieldConfig`

Reply:
Updated